package.json

```diff
@@ -64,7 +64,7 @@
 		"test": "./node_modules/jest-24.9.0/bin/jest.js --config tests/js/jest.config.json",
 		"test-staged": "./node_modules/jest-24.9.0/bin/jest.js --bail --config tests/js/jest.config.json --findRelatedTests",
 		"test:help": "wp-scripts test-unit-js --help",
-		"test:php": "docker-compose -f docker/wc-admin-php-test-suite/docker-compose.yml run --rm phpunit",
+		"test:php": "docker-compose -f docker/wc-admin-php-test-suite/docker-compose.yml run --rm phpunit ; docker-compose -f docker/wc-admin-php-test-suite/docker-compose.yml down",
```

Comment:
The `docker-compose run` command seems to only remove the `phpunit` container, but leaves the `mysql` container running.
Which is why I add the `docker-composer down`, which kills and removes the container (this is run even if the phpunit tests fail).